### ISSUE TITLE
Flexible Layouts display Condition Sets as their editing/browsing interface

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#4141]
Condition Sets appear in Flexible Layouts as either their main view browse or edit interfaces, depending on the current edit mode. Instead, they should behave as in Display Layouts and appear as telemetry, with alphanumeric, overlay and stacked plot view options. Given that we don't have type switching in Flexible Layouts for numeric or enum telemetry, just displaying as an alphanumeric as a V1 fix would be better than the current.

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [ ] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug?

#### Steps to Reproduce
1. Add a Condition Set into a Flexible Layout.

#### Environment
* Open MCT Version: 1.7.6
* Deployment Type: stable
* OS:
* Browser:

#### Additional Information
Current
![image](https://user-images.githubusercontent.com/1056412/130531417-163141ba-696f-46af-ba1a-a2df8e446014.png)
![image](https://user-images.githubusercontent.com/1056412/130531431-6881773e-ea7e-4e86-afa6-003037023217.png)

Desired
![image](https://user-images.githubusercontent.com/1056412/130531570-5743183d-8604-4eec-9ddd-2a9396325f2c.png)

**Comment by shefalijoshi** [comment]
@charlesh88 Getting an alphanumeric view for condition sets in the flexible layout is rather involved.
For now we could go with a Plot or Table view. Is that acceptable?

**Comment by akhenry** [comment]
Yeah, there are two issues here. The most pressing issue is that Condition Sets are not being treated the same as other telemetry points, which is an easy fix. We probably just need to add some logic to `canView` and `canEdit` on the `ConditionSetViewProvider` that uses the optional `objectPath` argument and returns false if the condition set is embedded.

The second issue is that alphanumeric views are not supported for any telemetry types in flex layouts. Unfortunately, as @shefalijoshi mentioned, fixing this is quite involved. Alphanumeric views are deeply tied to Display Layouts and were unfortunately never implemented as independent views.

I think the right thing to do initially is to make sure Condition Sets display like any other telemetry in a flex layout.

**Comment by akhenry** [comment]
Vacillating on the severity here 😄 Initially changed to blocker on grounds of data loss, but reverted because ultimately the output of the CS is still **technically** visible.

**Comment by shefalijoshi** [comment]
Verified Fixed.
